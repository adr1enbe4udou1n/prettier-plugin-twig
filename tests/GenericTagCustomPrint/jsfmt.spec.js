run_spec(__dirname, ["melody"], {
    twigMultiTags: ["switch,case,default,endswitch"],
    twigMelodyPlugins: ["src/plugins/switch-plugin"]
});
