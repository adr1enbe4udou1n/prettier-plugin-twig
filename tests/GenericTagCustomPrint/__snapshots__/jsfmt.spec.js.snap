// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`switch.melody.twig - melody-verify > switch.melody.twig 1`] = `
{% switch matrixBlock.type %}
    {% case "text" %}

        {{ matrixBlock.textField | markdown }}

    {% case "image" %}

        {{ matrixBlock.image[0].getImg() }}

    {% default %}

        <p>A font walks into a bar.</p>
        <p>The bartender says, “Hey, we don’t serve your type in here!”</p>

{% endswitch %}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{% switch matrixBlock.type %}
    {% case 'text' %}
        {{ matrixBlock.textField|markdown }}
    {% case 'image' %}
        {{ matrixBlock.image[0].getImg() }}
    {% default %}
        <p>
            A font walks into a bar.
        </p>
        <p>
            The bartender says, “Hey, we don’t serve your type in here!”
        </p>
{% endswitch %}

`;
